core/storage/feedback/gae_models.py

```diff
@@ -179,21 +251,44 @@ def create(cls, exploration_id, thread_id, message_id):
 
     @classmethod
     def get(cls, exploration_id, thread_id, message_id, strict=True):
-        """Gets the FeedbackMessageModel entry for the given ID.
-
-        If the message id is valid and it is not marked as deleted, returns the
-        message instance. Otherwise:
-        - if strict is True, raises EntityNotFoundError
-        - if strict is False, returns None.
+        """Gets the FeedbackMessageModel entry for the given ID. Raises error if
+        no undeleted messages are found and strict == True.
+
+        Args:
+            exploration_id: str. ID of the exploration.
+            thread_id: str. ID of the thread.
+            message_id: int. ID of the message.
+            strict: bool. Whether to raise an error if no FeedbackMessageModel
+                entry is found for the given IDs.
+
+        Returns:
+            FeedbackMessageModel or None. If strict == False and no undeleted
+                message with the given message_id exists in the datastore.
+                Otherwise the FeedbackMessageModel instance that corresponds
+                to the given ID.
+
+        Raises:
+            EntityNotFoundError: If strict == True and message ID is not valid
+                and it is not marked as deleted. No error will be raised if
+                message is deleted.
```

Comment:
Can you please explain how to use interactive console after setting deleted=True?